scipy/stats/_distn_infrastructure.py

```diff
@@ -2266,6 +2313,10 @@ def fit(self, data, *args, **kwds):
         >>> loc1, scale1
         (0.92087172783841631, 2.0015750750324668)
         """
+
+        # memory for method of moments
```

Comment:
So the moments only need to be calculated once per `fit`, not each iteration. Save as an attribute of the class to minimize changes to existing code (passing around as an argument).

Reply:
We decided to pass it around as an argument.